include/itkStructurePreservingColorNormalizationFilter.hxx

```diff
@@ -118,79 +118,79 @@ StructurePreservingColorNormalizationFilter< TImage >
   itkAssertOrThrowMacro( m_ColorIndexSuppressedByHematoxylin >= 0 && m_ColorIndexSuppressedByEosin >= 0,
     "Need to set ColorIndexSuppressedByHematoxylin and ColorIndexSuppressedByEosin before using StructurePreservingColorNormalizationFilter" );
 
-  // Find input and refer and make iterators for them.
-  const ImageType * const input = this->GetInput( 0 ); // image to be normalized
-  const ImageType * const refer = this->GetInput( 1 ); // reference image
+  // Find inputImage and referenceImage and make iterators for them.
+  const ImageType * const inputImage = this->GetInput( 0 ); // image to be normalized
+  const ImageType * const referenceImage = this->GetInput( 1 ); // reference image
   // For each of the two images, make sure that it was supplied, or
   // that we have it cached already.
-  itkAssertOrThrowMacro( input != nullptr, "An image to be normalized needs to be supplied as input image #0" );
-  itkAssertOrThrowMacro( refer != nullptr || m_refer != nullptr, "A reference image needs to be supplied as input image #1" );
+  itkAssertOrThrowMacro( inputImage != nullptr, "An image to be normalized needs to be supplied as input image #0" );
+  itkAssertOrThrowMacro( referenceImage != nullptr || m_Reference != nullptr, "A reference image needs to be supplied as input image #1" );
 
   // For each image, if there is a supplied image and it is different
   // from what we have cached then compute stuff and cache the
   // results.  These two calls to ImageToNMF could be done
   // simultaneously.
-  RegionConstIterator inputIter {input, input->GetRequestedRegion()};
+  RegionConstIterator inputIterator {inputImage, inputImage->GetRequestedRegion()};
```

Comment:
Iterators are usually highly local and participate in a lot of expressions so their names are usually kept short, e.g. `inIt`, `refIt`.

Reply:
Done.